### PR TITLE
pdftotext: add page

### LINKED ISSUE #1736: page request: pdftotext
Would nicely complement the existing [pdftk.md](https://github.com/tldr-pages/tldr/blob/master/pages/common/pdftk.md) page and other pdf-related commands.

Relevant links: [Wikipedia article](https://en.wikipedia.org/wiki/Pdftotext), [man page](https://linux.die.net/man/1/pdftotext).

### PR DESCRIPTION
**Pull request by redant333**
Fixes #1736

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

**Comment by redant333** [comment]
This will resolve #1736

File: pages/common/pdftotext.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+- Convert filename.pdf to plain text and save it as filename.txt:
+
+`pdftotext {{filename.pdf}}`
+
+- Convert pdf-file.pdf to plain text and save it as text-file.txt:
+
+`pdftotext {{pdf-file.pdf}} {{text-file.txt}}`
+
+- Convert pdf-file.pdf to plain text and print it to standard output:
```
Comment (agnivade) [inline pages/common/pdftotext.md]: Make this the first example. Change file to {{filename.pdf}}

File: pages/common/pdftotext.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# pdftotext
+
+> Convert PDF files to plain text format.
+
+- Convert filename.pdf to plain text and save it as filename.txt:
+
+`pdftotext {{filename.pdf}}`
+
+- Convert pdf-file.pdf to plain text and save it as text-file.txt:
```
Comment (agnivade) [inline pages/common/pdftotext.md]: Change the files to input.pdf and output.txt

File: pages/common/pdftotext.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+`pdftotext {{pdf-file.pdf}} {{text-file.txt}}`
+
+- Convert pdf-file.pdf to plain text and print it to standard output:
+
+`pdftotext {{pdf-file.pdf}} -`
+
+- Convert pages 2, 3 and 4 of pdf-file.pdf to plain text and print them to standard output (parameters -f and -l can also be used with previous variants):
```
Comment (agnivade) [inline pages/common/pdftotext.md]: Use this as a save to text file example. And no need to mention that `-f` and `-l` can be used with previous examples. That is implicit and in-fact how our pages are structured.

And does the command support expanded options for `-f` and `-l` ?

Comment (redant333) [reply pages/common/pdftotext.md]: No expanded options, just the short ones.

**Comment by redant333** [comment]
Updated per comments.

**Comment by sbrl** [comment]
Thanks, @redant333!

**Pull request merged**